### ISSUE TITLE
Add support for exposing fractal-web on `example.org/subpath`

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-web#561]
This will be needed if we want to maintain two different Fractal deployments on the same domain, e.g.
* fractal.example.org
* fractal.example.org/somelegacyversionthatshouldbedeprecatedsoon

This would require some additional apache configuration (which should pose no problem), and some additional support in fractal-web (currently lacking).

Perhaps it has to do with some base URL to be added _somewhere_ - TBD.

**Comment by zonia3000** [comment]
**TL;DR**: it is possible to specify a base path only at build time (so we need to rebuild the frontend to support a different base path); as a consequence we can't use the result of the CI build for deployments needing a custom base path.

The standard way to set the base path in Svelte is by defining it `svelte.config.js`:

```js
const config = {
  kit: {
    paths: {
      base: '/subpath'
    }
  }
}
```

And then update every URL to use it:

```html
<script>
import { base } from '$app/paths';
</script>

<a href="{base}/your-page">Link</a>
```

See: https://kit.svelte.dev/docs/configuration#paths

A workaround that I tried is to set an empty base path together with the relative flag. The idea was to also add a custom environment variable (that could be set at runtime) to use as our dynamic base path.

```js
const config = {
  kit: {
    paths: {
      base: '',
      relative: true
    }
  }
}
```

The result of this is that Svelte assets paths are created as relative paths:

```html
<link href="./_app/immutable/assets/0.DEK9edj8.css" rel="stylesheet">
```

This could work with the following Apache configuration:

```
<Location /subpath>
    ProxyPass http://localhost:5173
    ProxyPassReverse http://localhost:5173
</Location>
```

But it seems to require a final slash: http://localhost/subpath doesn't work, while http://localhost/subpath/ works. This could be enforced by further Apache configuration but it doesn't look very robust.

**Comment by tcompa** [comment]
Thanks a lot for checking this out. As we briefly discussed, my take is that this is not worth it for specific goal we had in mind - more on this later.
